HousingSearchApi.Tests/V1/E2ETests/Stories/GetAssetStories.cs

```diff
@@ -131,5 +131,15 @@ public void ServiceReturnsExactMatchOnlyWhenIsFilteredQueryTrue()
                 .Then(t => _steps.ThenThatAddressShouldBeTheOnlyResult("N1 6TY"))
                 .BDDfy();
         }
+
+        [Fact]
+        public void ServiceFiltersGivenContractIsApprovedStatusWithoutSearchText()
```

Comment:
Because the `assetContract.isApproved` filter is added to all queries in the query generator would be good to cover the two additional scenarios too to ensure the results are correct:
1. Results when `contractIsApproved` is provided and set to `true`
2. Results when `contractIsApproved` is not provided at all. From query builders point of view this should default to `false`, so how will that affect the results?

Reply:
Addressed in [360a646](https://github.com/LBHackney-IT/housing-search-api/pull/231/commits/360a6465b1a887c892371429b4cb6b33d73a4ad9).